### PR TITLE
Update functional_chaos.rst

### PR DESCRIPTION
**Pull request by mbaudin47**
Update Functional Chaos Expansion theory :

http://openturns.github.io/openturns/latest/theory/meta_modeling/functional_chaos.html

image

### CODE REVIEW

File: python/doc/theory/meta_modeling/functional_chaos.rst [hunk 1]
```diff
@@ -3,68 +3,71 @@
...
-  all types of response surface that can be seen as a projection of the
-  physical model in an orthonormal basis. This class of methods uses the
-  Hilbertian space (square-integrable space: :math:`L^2`) to construct
-  the response surface.
-| Assuming that the physical model has a finite second order measure
-  (i.e. :math:`E\left( \|h(\underline{X})\|^2\right)< + \infty`), it may
-  be uniquely represented as a converging series onto an orthonormal
-  basis as follows:
+Accounting for the joint probability density function (PDF)
+:math:`f_{\underline{X}}(\underline{x})` of the input random vector
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/functional_chaos.rst:7]: While you are at it, you may replace all the `\underline` commands with the `\vect` macro.

Comment (mbaudin47) [reply python/doc/theory/meta_modeling/functional_chaos.rst]: Fixed at 714295c11

File: python/doc/theory/meta_modeling/functional_chaos.rst [hunk 2]
```diff
@@ -3,72 +3,75 @@
...
+:math:`\vect{Y}` is lost.
+A possible solution to overcome this problem and thereby to reduce the
+computational cost of MC simulation is to represent the random
+response :math:`\vect{Y}` in a suitable functional space, such as
+the Hilbert space :math:`L^2` of square-integrable functions with
+respect to the PDF :math:`f_{\vect{X}}(\vect{x})`.
+Precisely, an expansion of the model response onto an orthonormal
+basis of :math:`L^2` is of interest.
+
+The principles of the building of a (infinite numerable) basis of this
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/functional_chaos.rst:32]:
```suggestion
The principles of the building of an infinite countable basis of this
```

File: python/doc/theory/meta_modeling/functional_chaos.rst [hunk 3]
```diff
@@ -3,72 +3,75 @@
...
+space, i.e. the PC basis, are described in the sequel.
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/functional_chaos.rst:33]:
```suggestion
space, i.e. the PC basis, are hereafter described.
```

File: python/doc/theory/meta_modeling/functional_chaos.rst [hunk 4]
```diff
@@ -3,72 +3,75 @@
...
+
+Principle of the functional chaos expansion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider a model :math:`h` depending on a set of *random* variables
+:math:`\vect{X} = (X_1,\dots,X_{n_X})^{\textsf{T}}`. We call
+functional chaos expansion the class of spectral methods which gathers
+all types of response surface that can be seen as a projection of the
+physical model in an orthonormal basis. This class of methods uses the
+Hilbertian space (square-integrable space: :math:`L^2`) to construct
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/functional_chaos.rst:43]:
```suggestion
physical model in an orthonormal basis. This class of methods uses the
square-integrable (:math:`L^2`) Hilbertian space to construct
```

File: python/doc/theory/meta_modeling/functional_chaos.rst [hunk 5]
```diff
@@ -3,72 +3,75 @@
...
 
-  where the
-  :math:`\underline{y}_{i} = (y_{i,1},\dots,y_{i,n_Y})^{\textsf{T}}`\ ’s
-  are deterministic vectors that fully characterize the random vector
-  :math:`\underline{Y}`, and the :math:`\Phi_{i}`\ ’s are given basis
-  functions (e.g. orthonormal polynomials, wavelets).
+where the
+:math:`\vect{y}_{i} = (y_{i,1},\dots,y_{i,n_Y})^{\textsf{T}}`\ ’s
+are deterministic vectors that fully characterize the random vector
+:math:`\vect{Y}`, and the :math:`\Phi_{i}`\ ’s are given basis
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/functional_chaos.rst:57]:
```suggestion
:math:`\vect{y}_{i} = (y_{i,1},\dots,y_{i,n_Y})^{\textsf{T}}`
are deterministic vectors that fully characterize the random vector
:math:`\vect{Y}`, and the :math:`\Phi_{i}` are given basis
```

File: python/doc/theory/meta_modeling/functional_chaos.rst [hunk 6]
```diff
@@ -3,72 +3,75 @@
...
+functions (e.g. orthonormal polynomials, wavelets).
 
-| The orthonormality property of the functional chaos basis reads:
+The orthonormality property of the functional chaos basis reads:
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/functional_chaos.rst:60]:
```suggestion
The orthonormality property of the functional chaos basis can be written:
```

File: python/doc/theory/meta_modeling/functional_chaos.rst [hunk 7]
```diff
@@ -3,72 +3,75 @@
...
 
   .. math::
 
-     \langle \Phi_{i},\Phi_{j}\rangle = \int_{D}\Phi_{i}(\underline{x}) \Phi_{j}(\underline{x})~f_{\underline{X}}(\underline{x}) d \underline{x} = \delta_{i,j}.
+     \langle \Phi_{i},\Phi_{j}\rangle = \int_{D}\Phi_{i}(\vect{x}) \Phi_{j}(\vect{x})~f_{\vect{X}}(\vect{x}) d \vect{x} = \delta_{i,j}.
```
Comment (josephmure) [inline python/doc/theory/meta_modeling/functional_chaos.rst:64]:
```suggestion
     \langle \Phi_{i},\Phi_{j}\rangle = \int_{D}\Phi_{i}(\vect{x}) \Phi_{j}(\vect{x})~f_{\vect{X}}(\vect{x}) d \vect{x} = \delta_{i,j}
```

**Comment by mbaudin47** [comment]
This is a duplicate in PR #2421

**Pull request closed without merging**